ios/tmp.xcconfig

```diff
@@ -1 +1,2 @@
 ENV=STAGING
+SHOW_STORYBOOK=true
```

Comment:
Add this file to gitignore

Reply:
I have added it to .gitignore file and it is still adding it to commit. As discussed i will find the command and i will delete this file.